Haulage/AdminPage.xaml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Haulage.AdminPage"
+             Title="Admin"
+             Shell.NavBarIsVisible="False">
```

Comment:
this is set to false in a few places. Would it be worth removing lines 5 and 6 here as the title isn't being used and is just being set to be indivisible